CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
Is it really necessary?

Reply:
I thought that add_custom_command was added in 3.0.2, my bad.